package.json

```diff
@@ -40,34 +47,35 @@
     "@types/rsvp": "^4.0.1",
     "broccoli-asset-rev": "^3.0.0",
     "documentation": "^9.1.1",
-    "ember-ajax": "^4.0.0",
     "ember-cli": "~3.8.0",
-    "ember-cli-dependency-checker": "^3.0.0",
+    "ember-cli-dependency-checker": "^3.1.0",
     "ember-cli-htmlbars": "^3.0.0",
     "ember-cli-htmlbars-inline-precompile": "^2.0.0",
     "ember-cli-inject-live-reload": "^2.0.1",
-    "ember-cli-release": "^0.2.9",
-    "ember-cli-shims": "^1.1.0",
     "ember-cli-sri": "^2.1.0",
+    "ember-cli-template-lint": "^1.0.0-beta.1",
```

Comment:
I'd _vaguely_ prefer to differ from the blueprint here, and just add `ember-template-lint` directly...

Reply:
Ya I'm fine with this.